dist/tools/zep_dispatch/main.c

```diff
@@ -125,9 +145,42 @@ static void _info_handler(int signal)
     }
 }
 
+/* open mac802154_hwsim device to send frames to it */
+static int _open_mac802154_hwsim(const char *iface)
+{
+    int res;
+    int fd = socket(AF_PACKET, SOCK_RAW, htons(ETH_P_IEEE802154));
+
+    /* get interface index for interface name */
+    struct ifreq ifr;
+    strncpy(ifr.ifr_name, iface, IFNAMSIZ);
+
+    if ((res = ioctl(fd, SIOCGIFINDEX, &ifr)) < 0) {
+        goto error;
+    }
+
+    /* bind socket to the device */
+    struct sockaddr_ll sll = {
+        .sll_family = AF_PACKET,
+        .sll_ifindex = ifr.ifr_ifindex,
+        .sll_protocol = htons(ETH_P_IEEE802154),
+    };
+
+    if ((res = bind(fd, (struct sockaddr *)&sll, sizeof(sll))) < 0) {
+        goto error;
+    }
+
+    return fd;
+
+error:
+    close(fd);
+    fprintf(stderr, "open mac802154_hwsim: %s\n", strerror(res));
+    return res;
+}
+
 static void _print_help(const char *progname)
 {
-    fprintf(stderr, "usage: %s [-t topology] [-s seed] [-g graphviz_out] <address> <port>\n",
+    fprintf(stderr, "usage: %s [-t topology] [-s seed] [-g graphviz_out] [-w interface] <address> <port>\n",
```

Comment:
```suggestion
    fprintf(stderr, "usage: %s [-t topology] [-s seed] "
                    "[-g graphviz_out] [-w interface] <address> <port>\n",
```

Let's get rid of those nasty "line is longer than 100 chars" CI message